modules/roles/main.tf

```diff
@@ -12,36 +12,10 @@ resource "azurerm_role_assignment" "graphdb_backup" {
   role_definition_name = "Storage Blob Data Contributor"
 }
 
-resource "azurerm_role_definition" "managed_disk_manager" {
-  name        = "${var.resource_name_prefix}-ManagedDiskManager"
-  scope       = var.resource_group_id
-  description = "This is a custom role created via Terraform required for creating data disks for GraphDB"
-
-  permissions {
-    actions = [
-      "Microsoft.Compute/disks/read",
-      "Microsoft.Compute/disks/write",
-      "Microsoft.Compute/virtualMachineScaleSets/read",
-      "Microsoft.Compute/virtualMachineScaleSets/virtualMachines/write",
-      "Microsoft.Compute/virtualMachineScaleSets/virtualMachines/read",
-      "Microsoft.Network/virtualNetworks/subnets/join/action",
-      "Microsoft.Network/applicationGateways/backendAddressPools/join/action",
-      "Microsoft.Network/networkSecurityGroups/join/action"
-    ]
-    not_actions = []
-  }
-
-  assignable_scopes = [
-    var.resource_group_id
-  ]
-}
-
 resource "azurerm_role_assignment" "rg_contributor_role" {
```

Comment:
Update the resource name to match the role.

Reply:
updated